_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  It's possible to avoid this problem, but it requires that the
+  two-party ECDSA protocol be designed to include an extra commitment
+  procedure, which Nick notes some two-party ECDSA protocols and
+  implementations already do.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why was the current formula that calculates 'target' from 'nBits' chosen?]({{bse}}96298)
+  Ravi Patel asks why a simpler formula for calculating the difficulty target
+  from `nBits` wasn’t chosen. Andrew Chow dives into some of the details around
+  the formula, its history, and even sample code from Bitcoin’s 0.1.5 release.
+
+- [Does Bitcoin really need timestamps?]({{bse}}96185)
+  Pieter Wuille explains why limiting the block rate without reference to a
+  clock time outside of the blockchain could make running full nodes more
+  expensive while also struggling to keep the stale block rate low and
+  preventing collusion attacks.
+
+- [In a fee overpayment attack, why can't compromised software provide fake previous transactions corresponding to fake inputs?]({{bse}}96309)
+  Regarding a [fee overpayment attack][news101 fee overpayment attack] on segwit
+  transactions with multiple inputs, justinmoon asks why the remedy of the
+  attack, requiring copies of previous transactions for the inputs, is not
+  vulnerable to malicious software providing fake previous transactions. Since
+  any provided previous transaction must have a hash that matches the spending input’s
+  previous transaction hash, such an attack is not feasible.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta] this release candidate for an
+  LND maintenance release is now available for testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19260][] p2p: disconnect peers that send filterclear + update existing filter msg disconnect logic FIXME:dongcarl
+
+- [Bitcoin Core #19133][] restores the bitcoin-cli `generate` command
```

Comment:
"restores" isn't right here, since there was never a bitcoin-cli generate command previously. More accurate would be: "Bitcoin Core 19133 adds a bitcoin-cli `generate` alias to replace the functionality of the bitcoind `generate` RPC method, which was removed from Bitcoin Core in V..."